project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+
+
+      - &HP_cooling_efficiency_ducted
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+        apply_logic:
+          - or:
+            - *dualfuel_ASHP_no_ducted_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_FO_furnace_apply_logic
+
+
+
+
+      - &100_percent_conditioned_ducted
+        option: HVAC Cooling Partial Space Conditioning|100% Conditioned
+        apply_logic:
+          - or:
+            - *dualfuel_ASHP_no_ducted_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_FO_furnace_apply_logic
+
+
+
+
+
+
+  - upgrade_name: EnergyStar cold climate ducted ASHP w/ elec backup  #also provides SEER 15 cooling measure profile
+    # Measure 2
+    options:
+      #ASHP
+      - &ducted_ASHP_EnergyStarCC_exist_backup_no_shared_ducts
+        option: HVAC Heating Efficiency|ASHP, SEER 16, 9.5 HSPF, CCHP, Separate Backup # same as base option
+        apply_logic: *dualfuel_ASHP_no_ducted_furnace_apply_logic
+        costs: &costs_ashp_ducted_SEER_16_9-5_HSPF
+        # Source: Custom regression by Brennan Less. Ducted heat pump project costs were regressed on nameplate tons and HSPF,
+        # from data described in in Less et al. https://doi.org/10.20357/B7FP4D. 2019$ inflated to 2022$ using 1.12 factor.
+        # Ducted ASHP regression n=317. Implicitly includes electrical upgrades and electric backup.
+          - value: 3907.01 # for HSPF 9
+            multiplier: Fixed (1)
+          - value: 155.17
+            multiplier: Size, Heating System Primary (kBtu/h)
+        lifetime: &lifetime_ashp_ducted_SEER_16_9-5_HSPF 15
+
+
+      - *HP_cooling_efficiency_ducted
```

Comment:
I think it's cleaner to have things in the same order in each upgrade. These two global changes were at the end of upgrade 1 and are at the beginning (or nearly) here - standardize on one or the other.